common.go

```diff
@@ -17,3 +18,20 @@ func debugLog(format string, args ...interface{}) {
 	}
 	log.Printf(format+"\n", args...)
 }
+
+//logURL replaces existing password from the given URL
+func logURL(URL string) (string, error) {
+	// Hide password if set in URL
+	u, err := url.Parse(URL)
+	if err != nil {
+		return "", err
+	}
+	if u.User != nil {
+		_, exists := u.User.Password()
+		if exists {
```

Comment:
Combine the above two lines:

if _, ok := u.User.Password(); ok {

(Just use "ok", which is a common idiom for Go map presence variables).